rgw/v2/tests/s3cmd/test_s3cmd.py

```diff
@@ -209,21 +214,31 @@ def test_exec(config, ssh_con):
         # Create a bucket
         s3cmd_reusable.create_bucket(bucket_name)
         log.info(f"Bucket {bucket_name} created")
+        object_count = config.objects_count
 
-        # Upload file to bucket
-        uploaded_file_info = s3cmd_reusable.upload_file(
-            bucket_name, test_data_path=TEST_DATA_PATH
-        )
-        uploaded_file = uploaded_file_info["name"]
-        log.info(f"Uploaded file {uploaded_file} to bucket {bucket_name}")
+        if object_count > 500:
```

Comment:
@viduship , This method would error out in case there is another single site config with 1K object uploads. Instead can we pass a test_ops here to execute only if parameter 'full_sync_at_archive'  is true.